docs/devsite-help/troubleshooting.md

```diff
@@ -218,3 +218,62 @@ native libraries are copied appropriately.
 
 This information is now in the [protocol buffers guide](protobuf.md#repeated-fields-and-maps); this section has been
 retained for the sake of existing links.
+
+## How can I get ErrorInfo from RpcException?
+
+`ErrorInfo` in `RpcException` provides a consistent and human readable error details.
+It contains three major fields - reason, domain and metadata described as below:
+
+- `Reason` - It is a constant value and indicates the proximate cause of 
+the error. Error reasons are unique within a particular domain of errors.
+
+- `Domain` - It is the logical grouping to which the error belongs. It is 
+typically the registered service name of the tool or product that 
+generates the error. Example: "pubsub.googleapis.com". If the error 
+is generated by some common infrastructure, the error domain must be 
+a globally unique value that identifies the infrastructure. 
+For Google API infrastructure, the error domain is "googleapis.com".
+
+- `Metadata`- It is the additional structured details about the error. 
+It can help in identifying the exact cause of the error.
+
+```csharp
+using Google.Api.Gax.Grpc;
+try
+{
+  // Code that can throw the RpcException.
+}
+catch (RpcException  ex)
+{
+   var errorInfo = ex.GetErrorInfo();   
+   var domain = errorInfo.Domain;
+   var reason = errorInfo.Reason;
+   var metadata = errorInfo.Metadata;
+
+   // Use the domain, reason and metadata of ErrorInfo, e.g. for logging or taking remedial action.
```

Comment:
What remedial action?
Only reason I say this is that without concrete examples, this may paint an incomplete picture and the reader may end up being curious about it with no link or reference to the additional information. I 'll let Jon decide on it so please don't make any changes for now.

Reply:
That's really application-specific and API-specific. I think it's fine to leave the comment as is (within the suggestion above).